modules/login/test/Request_Account_test_plan.md

```diff
@@ -8,7 +8,7 @@
 5. Verify "Return to Loris login page" link on Thank you page works
 	
 ### Approving new User Account Request:
-6. Log in as another user who has permission:user_accounts and does Not have permission:user_accounts_multisite.  Verify that new account request notification is counted in Dashboard (count has incremented).
+6. Log in as another user who has permission:user_accounts and does Not have permission:user_accounts_multisite (User Management and Across all sites create and edit users).  Verify that new account request notification is counted in Dashboard (count has incremented).
```

Comment:
- remove the double space before `Verify that new account`
- add space after `permission:` for clarity

Reply:
Hi, what do you mean by the last point.

Reply:
6. Log in as another user who has permission `user_accounts` _(insert frontend lable here)_ and does not have permission `user_accounts_multisite` (User Management and Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).

Reply:
Alright, thanks.